assets/src/js/admin/reports/components/chart/utils/index.js

```diff
@@ -10,7 +10,8 @@ export function formatData (type, data) {
 
         const formatted = {
             label: (' ' + dataset.label).slice(1),
-            data: dataset.data.slice(0),
+			data: dataset.data.slice(0),
+			yAxisID: 'y-axis-' + index,
```

Comment:
@henryholtgeerts Code is not formatted. Did you merge the `develop` branch?

Reply:
Merged develop and resolved conflicts, ref: https://github.com/impress-org/givewp/pull/4412/commits/e73dc75706873dfd0eba1239a0ecde619890e15c